src/otx/recipe/detection/rtmdet_tiny.yaml

```diff
@@ -208,3 +209,5 @@ model:
         debug: false
         pos_weight: -1
       type: RTMDet
+trainer:
+  precision: 32
```

Comment:
Is this to align with OTX 1.5? Or is it because it's a model that doesn't work?

Reply:
I actually have the same question, is there any specific motivation/decision for changing the default to 16?

Reply:
@samet-akcay That's a bit of a reverse question for me. By applying mixed precision, we can gain in terms of memory and speed. This is the approach we use in OTX1.5 as well.
I was actually wondering why this model couldn't use 16.

Reply:
oh, I've commented on the wrong file :)

Reply:
But it makes sense, thanks for the clarification

Reply:
This model should be trained by precision32 because in precision16, this model raises errors during training.
And this is a new model to OTX, that is we don't have RTMDet-tiny in otx1.x